specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/PlaybookConfigurations/PutPlaybookConfiguration_example.json

```diff
@@ -0,0 +1,175 @@
+                ]
+              }
+            ]
+          }
+        ],
+        "actions": [
+          {
+            "resourceId": "/subscriptions/e54a4a18-5b94-4f90-9471-bd3decad8a2e/resourceGroups/sample/providers/Microsoft.Logic/workflows/MyTest1",
+            "resourceProviderNamespace": "Microsoft.Logic",
+            "uri": "https://triggerUri1.com"
+          },
+          {
+            "resourceId": "/subscriptions/e54a4a18-5b94-4f90-9471-bd3decad8a2e/resourceGroups/sample/providers/Microsoft.Logic/workflows/MyTest2",
+            "resourceProviderNamespace": "Microsoft.Logic",
+            "uri": "https://triggerUri2.com"
+          }
+        ]
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Security/playbookConfigurations/myPlaybookConfigName",
+        "name": "myPlaybookConfigName",
+        "type": "Microsoft.Security/playbookConfigurations",
+        "location": "Central US",
+        "tags": {},
+        "properties": {
+          "description": "Alerts auto dismiss",
+          "isEnabled": true,
+          "metadata": {
+            "createdDateTimeUtc": "2019-03-27T08:43:54.5755094Z",
+            "createdBy": "2c429ed8-089f-42aa-9bb9-92312b8c9f72",
+            "lastUpdatedDateTimeUtc": "2019-03-28T08:43:54.5755094Z",
+            "lastUpdatedBy": "2c429ed8-089f-42aa-9bb9-92312b8c9f72"
+          },
+          "scopes": [
+            {
+              "description": "Some custom description",
+              "scopePath": "/subscriptions/6618f01b-9b61-4cfc-9010-836b34b1bb2a"
+            },
+            {
+              "description": "Some custom description",
+              "scopePath": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23"
+            }
+          ],
+          "sources": [
+            {
+              "eventSource": "AscAlerts",
+              "ruleSets": [
+                {
+                  "rules": [
+                    {
+                      "propertyJPath": "$.type",
+                      "propertyType": "System.String",
+                      "expectedValue": "customType",
+                      "operator": "EqualsIgnoreCase"
+                    }
+                  ]
+                }
+              ]
+            }
+          ],
+          "actions": [
+            {
+              "resourceId": "/subscriptions/e54a4a18-5b94-4f90-9471-bd3decad8a2e/resourceGroups/sample/providers/Microsoft.Logic/workflows/MyTest1",
+              "resourceProviderNamespace": "Microsoft.Logic",
+              "uri": "https://triggerUri1.com"
+            },
+            {
+              "resourceId": "/subscriptions/e54a4a18-5b94-4f90-9471-bd3decad8a2e/resourceGroups/sample/providers/Microsoft.Logic/workflows/MyTest2",
+              "resourceProviderNamespace": "Microsoft.Logic",
+              "uri": "https://triggerUri2.com"
+            }
+          ]
```

Comment:
A lot of this looks very similar to how Azure policy declares policy definitions. Have you looked at Azure policy at all?

Reply:
Yes of course. Much of this (and the language) was setup with Azure Policy (we are working on integration with them).